server/services/DBService.js

```diff
@@ -2,8 +2,8 @@ import Datastore from 'nedb';
 
 export default class DBService {
 
-  constructor() {
-    this.datastore = new Datastore({ filename: 'server/data.db', autoload: true });
+  constructor(conf) {
```

Comment:
Not really necessary to pass in the whole configuration object here. Please just use the `dbFilePath`

Reply:
Thanks for the feedback, fixed!